src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs

```diff
@@ -735,6 +710,31 @@ private Packaging.Core.PackageDependency CreateDependencyFromProject(dynamic pro
             }
         }
 
+        private static void ExtractMetadata(ProjectFactory projectFactory, Packaging.PackageBuilder builder)
```

Comment:
My bad, I didn't notice use of `projectFactory.ExtractMetadataFromProject`. 
As a suggestion can you make this method non-static then?